experimental/generation/generator/src/library/dialogGenerator.ts

```diff
@@ -686,9 +686,12 @@ export async function generate(
 
         // Expand schema expressions
         let expanded = expandSchema(schema.schema, scope, '', false, true, feedback)
-
+      
         // Write final schema
         let body = stringify(expanded, (key: any, val: any) => (key === '$templates' || key === '$requires' || key === '$templateDirs' || key === '$examples') ? undefined : val)
+        // Support integer type 
+        body = body.replace(/integer/g, 'number')
+
```

Comment:
This is not the right way to support this--specifics for given types should not be in the generator code. It should be that type: integer -> integer.lg template which defines a default $entities of number and brings in the required templates.